src/Microsoft.Identity.Web/MicrosoftIdentityOptions.cs

```diff
@@ -133,5 +134,19 @@ internal bool HasClientCredentials
         /// See https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/how-to-manage-ua-identity-portal.
         /// </summary>
         public string? UserAssignedManagedIdentityClientId { get; set; }
+
+        /// <summary>
+        /// Sets the ResetPassword route path.
```

Comment:
```suggestion
        /// ResetPassword path within the application's base path.
```

(proposing to remove the 'Sets' as it's set/get. Adding precision on what the path is.

Reply:
@Ponant are you able to update the comment as per @jmprieur 's suggestion? thx.